deploy/lib/ml_rest.rb

```diff
@@ -190,19 +158,19 @@ def install_extensions(path)
                 end
               end
             end
-
+          
           end
-
+          
           @logger.debug "extensionName: #{extensionName}"
           @logger.debug "headers: #{headers}"
           @logger.debug "params: #{params}"
-
-          url = "#{@rest_protocol}://#{@hostname}:#{@port}/v1/config/resources/#{extensionName}"
+          
+          url = "http://#{@hostname}:#{@port}/v1/config/resources/#{extensionName}"
           if (params.length > 0)
-            url << "?" << params.join("&")
+            url << "?name=#{extensionName}&" << params.join("&")
```

Comment:
Yes, agreed. Extension name should be part of the URL. I was experimenting with ways to fix this error.

"RESTAPI-INVALIDREQ: (err:FOER0000) Invalid request:  reason: cannot get resource extension without name".

I believe this is only an issue with hybrid apps.